consensus/istanbul/backend/backend.go

```diff
@@ -229,13 +229,28 @@ func (sb *backend) checkInSubList(prevHash common.Hash, valSet istanbul.Validato
 }
 
 // getTargetReceivers returns a map of nodes which need to receive a message
-func (sb *backend) getTargetReceivers(valSet istanbul.ValidatorSet) map[common.Address]bool {
+func (sb *backend) getTargetReceivers(prevHash common.Hash, valSet istanbul.ValidatorSet) map[common.Address]bool {
 	targets := make(map[common.Address]bool)
 
-	for _, val := range valSet.List() {
-		if val.Address() != sb.Address() {
-			targets[val.Address()] = true
+	cv := sb.currentView.Load().(*istanbul.View)
+	view := &istanbul.View{
+		Round:    big.NewInt(cv.Round.Int64()),
+		Sequence: big.NewInt(cv.Sequence.Int64()),
+	}
+
+	proposer := valSet.GetProposer()
+	for i := 0; i < 2; i++ {
+		vals := valSet.SubListWithProposer(prevHash, proposer.Address(), view)
```

Comment:
Don't we need to change the logic of SubListWithProposer()?

Reply:
@kjhman21 Do you mean adding round to previous hash? I tested the logic and the result is described in https://groundx.atlassian.net/browse/TPT-1024.

But I think it has to be applied in a different PR because it's scope is a bit different from this one.

Reply:
Ok, good. I was worried about a hard fork. This PR does not related to a hard fork. I will mark this as resolved. Thanks.